extras/iris_pipeline_project/data_folder/iris_test.csv

```diff
@@ -0,0 +1,2 @@
+SepalLength,SepalWidth,PetalLength,PetalWidth
```

Comment:
I guess we don't need that in the repo since we have an instruction on preparing the data.